src/wrapper-start.js

```diff
@@ -37,18 +37,11 @@
     var newErr;
     if (err instanceof Error) {
       var newErr = new Error(err.message, err.fileName, err.lineNumber);
-      if (isBrowser) {
-        newErr.message = err.message + '\n\t' + msg;
-        newErr.stack = err.stack;
-      }
-      else {
-        // node errors only look correct with the stack modified
-        newErr.message = err.message;
-        newErr.stack = err.stack + '\n\t' + msg;
-      }
+      newErr.message = msg + '\n\t' + err.message;
+      newErr.stack = err.stack;
```

Comment:
Otherwise the new error will report it's source as coming from withing addToError, which is misleading, even if it's strictly correct.